Movement generic standards

ryanscholzdvm opened the pull request:
As discussed in the February 2023 meeting, this pul request includes a handful of proposed additions to the standard to expand the existing eCVI standard to include generic traceability movements and sightings. These changes leave the existing eCVI document element unchanged, but create additional "Movement" and "Sighting" elements that make use of the existing standard elements and types.

ryanscholzdvm committed:
Formatting

Non-substantive formatting changes to standardize formatting of documentation and comments

ryanscholzdvm committed:
Created ApprovalType

Created Approval element and ApprovalType complex type to include a Person (approver), type of approval (enumerated list), and date of approval.

ryanscholzdvm committed:
NPIP 9-3 Structure

Added example implementation to support NPIP 9-3 Forms

ryanscholzdvm committed:
Create Generic Sighting Document

Create a generic "Sighting" document type to report location and animals with official identification.

ryanscholzdvm committed:
Revert "NPIP 9-3 Structure"

This reverts commit 379e7e1bfe6998ee4d62fc2a01e1aae50dcab027.

ryanscholzdvm committed:
Revert "Create Generic Sighting Document"

This reverts commit c55ce73c955cefe301603226e9426b198f59b11b.

ryanscholzdvm committed:
Created generic sighting element

Created a generic traceability sighting element for use in reporting a sighting event without associated movement.

ryanscholzdvm committed:
Create Generic Movement Element

Create a generic Movement element, based on the eCVI element, but with less restrictive use of Veterinarian, and altered attributes.

ryanscholzdvm committed:
Added US-Ship Examples

Added US-SHIP movement record examples utilizing the proposed movement document element. One example follows the traceability standard strictly (PIN and state for origin and destination). The second example uses a geopoint for a destination with an unknown PIN.

ryanscholzdvm committed:
Add Approval to Movement Element

Added Approval element to movement element definition

ryanscholzdvm committed:
NPIP 9-3 Example

Added Example of XML data for NPIP 9-3 form. Included redacted 9-3 form used to create XML data (names/contact info changed in XML)

ryanscholzdvm committed:
Update changelog

ryanscholzdvm committed:
Merge branch 'PendingChanges' into Movement_GenericStandards

mkm1879 commented:
I like this concept in general. My only concern is about "ownership" of the alternate document contents. That could be managed by documentation I suppose; something like "NPIP 9-3 root element maintained by ...". Alternately, other groups including the eCVI standard by version leaves the harmonization to those groups. 

How much does this workgroup want to take responsibility for?

ryanscholzdvm committed:
Include XML version number #86

Include an XMLSchemaVersion attribute in the top-level movement and signting elements.

ryanscholzdvm committed:
Remove history to allow merge

ryanscholzdvm committed:
Merge branch 'PendingChanges' into Movement_GenericStandards
